components/centraldashboard/app/api_workgroup.ts

```diff
@@ -27,6 +27,7 @@ interface HasWorkgroupResponse {
     user: string;
```

Comment:
Please make sure that the [***Backend Tests here***](https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/app/api_workgroup_test.ts) are updated and pass

Reply:
Done.

Reply:
Done.